lacework_node.tf

```diff
@@ -268,6 +270,10 @@ resource "kubernetes_daemonset" "lacework_datacollector" {
               key  = "config.json"
               path = "config.json"
             }
+            items {
```

Comment:
Could you add a new Terraform variable to toggle syscall enabled / disabled and gate the inclusion of the syscall config with this toggle?

Reply:
After many many tentatives, I found a way to make it work using an array and a dynamic block.